README.MD

```diff
@@ -2,7 +2,7 @@
 
 [![NuGet preview version](https://img.shields.io/nuget/vpre/ModelContextProtocol.svg)](https://www.nuget.org/packages/ModelContextProtocol/absoluteLatest)
 
-The official C# SDK for the [Model Context Protocol](https://modelcontextprotocol.io/), enabling .NET applications, services, and libraries to implement and interact with MCP clients and servers.
+The official C# SDK for the [Model Context Protocol](https://modelcontextprotocol.io/), enabling .NET applications, services, and libraries to implement and interact with MCP clients and servers. Please visit our [API documentation](https://modelcontextprotocol.github.io/csharp-sdk/api/ModelContextProtocol.html) for more details on available functionality.
```

Comment:
should we have a link to the landing page? https://modelcontextprotocol.github.io/csharp-sdk

Reply:
It doesn't have a lot content currently, the website has only been generated for hosting API documentation. Our goal is to integrate conceptual documentation with the official MCP website eventually.